### DISCUSSION TITLE
Increasing the Efforts for Greater Critical Cartography + Participatory Mapping

### DISCUSSION THREAD

**Discussion by jason-muljadi** [albertkun/21S-ASIAAM-191A#76] [Past Thinking Caps]
### Summary
In a nutshell, participatory mapping and critical cartography are essential tools in a technological society that allows others to connect based on data points. It is imperative that we look at the data that we receive and the agenda of the mapmakers/cartographers with every map/graph that they make. Maps are a geopolitical force that changes societies so it is with these tools that we can combat the forces of imperialism that lie in the world today.

### Part 1 - Understanding Participatory Mapping and Critical Cartography: The California Example
Maps have often been used to exclude individuals or groups from what people deem as the "civilized world." The Anglo viewpoint has always been the predominant viewpoint in the maps were used to chart territories, as exemplified by the early British colonial settlement maps and the erasure of Native American lands **(Cosgrove, 2008)**. As an example... 

![image](https://user-images.githubusercontent.com/81675803/114329073-56449e00-9af3-11eb-97f1-4c203aea4e58.png)

This is a Google Maps image of part of the state of California. What do you see and what do you not see? Compare this with another visualization of the same landmass.

![image](https://user-images.githubusercontent.com/81675803/114329102-62c8f680-9af3-11eb-88e2-4649b5a2640e.png)
It is jarring to see the discrepancy and the erasure of Native American lands that are hard to find in the maps that we use today. Could we tell from the Google Maps images above that there was a California Indian genocide in the 1800s or that the population of American Indians was reduced from hundreds of thousands to a few thousands? Currently, there are approximately 300k Native Americans who live in California, but do the maps that we use every day show us this true history? What happened to these territories once held by the many tribes of California? If the land is still the same, who is it that holds power over the land? Colonial violence persists through our perception of maps through the erasure of the history of indigenous folks and the commoditization of the land on which we stand on.

Without the work from Native American communities, we would be unable to visualize what lands did the Indigenous people of this land had pre-contact European. The erasure of history and culture is painful, and we keep the memories of truth alive through the process of critical cartography and working with Native American communities. 

### Part 2 - Expanding Beyond Geography
It feels that the idea of a map can somewhat be stagnant for most individuals. Usually, mapping would follow a Cartesian coordinate plot followed by indicating where an object is through the different axis and linear equations. 

![image](https://user-images.githubusercontent.com/81675803/114330250-277bf700-9af6-11eb-8b3c-dbf4a2942880.png)
![image](https://user-images.githubusercontent.com/81675803/114330283-382c6d00-9af6-11eb-9ddf-acc739e7491f.png)
![image](https://user-images.githubusercontent.com/81675803/114330321-4d090080-9af6-11eb-9137-20a498cb3630.png)
**(Wilmot 2020)**

Transposing cartography into the digital age means that there needs to be an ample way to format how we envision the world through the technology of today. With satellite images from companies such as Google, code allows us to map the world in different ways in order for us to see it through different lenses. Part of critical cartography is understanding the modern techniques used to envision the world, especially when it comes to computer programming and selecting a color component. Even though it seems trivial, colors play a huge role in the way we perceive data, as color may have an innate effect on our brains that may skew the data. 

![image](https://user-images.githubusercontent.com/81675803/114331406-c6a1ee00-9af8-11eb-91be-bd56716eaaf0.png)
![image](https://user-images.githubusercontent.com/81675803/114331483-ee915180-9af8-11eb-80d4-44fae37e222c.png)
**(Wilmot 2020)**

It is important to also be critical in the way we present our data and that an aspect of critical cartography comes from a proficient skill set in how we digitally map locations and data out and acknowledging any or eliminating bias that comes even from theoretical methodologies. Math has always dictated the foundational structures of maps, but with new ways to visualize data, it is imperative to expand our intake of information through these new methodologies as well. 

Perhaps one of the ways to understand the world is through Robert Tally's model of _geocriticism._ Geocriticism is the idea that space is not an empty container and that all of the actors and events that happened in that space, regardless of time, contribute to understanding the geography of a landmass. This practice also incorporates the literature in the space itself. It is focused on the literary aspects of the work that the actors that resided in the place produced, such as novels, travel brochures, or even political documents **(Tally, 2019)**. 

### Part 3 - Participatory Mapping and Critical Cartography in Modern Lenses: Israel and Palestine
Geopolitical events have quite certainly shaped our worldly paradigms, especially when it comes to territorial claims. One of the most controversial cartographical challenges lies in the case of Palestine and Israel. Politicians and researchers both have argued for the "correct" representation of Israel and Palestine. Collaborative mapping for these Middle Eastern territories seems to be from elitist perspectives that have high variances in the way they perceive the two territories to be. 

OpenStreetMap (OSM) and WikiMapia have been powerful tools for both the Israeli and Palestinian camps. With the data they produce, they are able to attempt to showcase why one side should be favored over the other. The maps produced are often fragmented and messy like their relationship, and each data set should be analyzed carefully to understand the point of view that is being shown. 

![image](https://user-images.githubusercontent.com/81675803/114333882-3c5c8880-9afe-11eb-9333-1825b5b30e23.png)
![image](https://user-images.githubusercontent.com/81675803/114333868-39619800-9afe-11eb-93ef-73391f7c3e60.png)
**(Bittner & Glasze, 2021)**

It makes sense that there would be a difference of users in OSM and WikiMapia, and their geographical location and social status will definitely play a role in how territory is mapped. 

![image](https://user-images.githubusercontent.com/81675803/114334036-96f5e480-9afe-11eb-9c93-b7764dfae8af.png)
![image](https://user-images.githubusercontent.com/81675803/114334043-9a896b80-9afe-11eb-9443-8533c93ac844.png)
**(Bittner & Glasze, 2021)**

We see that the drawback of participatory mapping is that people are prone to abiding by their agendas. It is difficult to police these voluntary actions, which would defeat the purpose of participatory mapping. Personally, I do not have a solution when it comes to mapping the territories because of the bias that resides in the communities of the Palestine/Israel area. Perhaps the solution would be to "take the middle ground" of that data point, but that would also lead to the argument of: what exactly is the middle ground and who gets to define it?

### Part 4 - A Case Study of the Brazilian Community of Xavier
While critical cartography and participatory mapping are two tools that could allow greater democratization in society, there are always drawbacks to consider. Some of the drawbacks of critical cartography have been the issue of expenses and time itself. This can be seen in the participatory videographic research with the traditional Brazilian community of Xavier. Xavier, a community of 22 families with common housing, has been in conflict with the wind farms that the wind farmers have built near their community. To showcase the adverse effects of the wind farms, researchers created a 20-minute video filled with personal interviews and 2D/3D models of the area to showcase Xavier. At first, the video was not the main goal of the experiment, as the community leaders asked for a 2D/3D map instead. However, the researchers found a videographer who would donate their time to help with this project, therefore making the project much more indispensable and easier to read. The researchers also allocated time and resources to educate the people of Xavier of technologies so that they can participate in the making of the video **(Gorayeb et al. 2021)**. 

The benefits of Gorayeb et al. experiment was that it was able to encapsulate the need of a community through an efficient and precise technology that allowed for personal flavor. Traditional livelihoods in different areas of Brazil could benefit from this technology, as an international audience of all ages (skewed towards the younger generation) can learn and empathize with the struggles of the Xavier community. However, the video was not without controversy. The initial screening of the video to the Xavier community was filled with arguments over the lack of diverse representation in the video (only a few of the many folks that were interviewed were shown on the screen). The researchers believe it was because of the years of infighting that has been a part of the community ever since the wind farm. Yet, I would argue that the exclusion of individuals is the primary cause of the Xavier community's anger. Because the researchers themselves did not plan for a videographer, their project was not planned as efficiently as it could have been. There were also the issues of time, as they did not have the time to ensure that the product presented was aligned with the community's value. Therefore, the 20-minute video, "We Made Our Map: Territory, Participatory Cartography, and Wind Power in the West Coast of Ceará, Brazil" (LabocartUFC, 2016) shows how efforts to highlight a community's need is difficult and could actually upset the community. **_With all efforts to ensure participatory mapping, there needs to be enough time and resources to ensure that many, if not all, of the voices in the community, are heard and represented accurately._** 

### Part 5: Where Do We Go From Here?
Activism through critical cartography and participatory will never be perfect, but that is why it is imperative to try. Though there may be many setbacks that come from establishing a paradigm in which persons from all communities can understand, the greatest drawback would be to stay stagnant with mapping and not try. Miren Gutiérrez, in her work, provides three examples in which geoactivism can be shown in a new way that highlights the data points more effectively. 

![image](https://user-images.githubusercontent.com/81675803/114335120-f48b3080-9b00-11eb-8815-06ee080317b7.png)
![image](https://user-images.githubusercontent.com/81675803/114335122-f6ed8a80-9b00-11eb-825a-9cdfa6f8451f.png)
![image](https://user-images.githubusercontent.com/81675803/114335126-f8b74e00-9b00-11eb-829f-f3d5c47d0f86.png)
**(Gutiérrez, 2020)**

In the maps that she analyzed, she also analyzed the words that they were using to title the maps, stating that some maps were empowering or discomforting. For the maps: ‘Left-to-die boat’ and ‘Western Africa missing fish,’ it was clear that this data was meant to show something to condemn, whether it be a terrible administrative failure that led to a hostage situation or the threat of lacking food security due to the lack of fish. Yet, the 15M map showed the connection of Spain's indigin@d movement, which was such an empowering visualization to see. 

All of these maps show something so uniquely different and stretches beyond the ideologies of Western cartography. All of them had stories to be told that were extremely personal. Data points become people, and that is the ultimate goal to increase critical cartography and mindful participatory mapping. Our visualizations must not only map out the communities' goals, but also their faces and their stories that connect us to our shared bond of humanity.

### Part 6 - Sources:
Bittner, Christian, and Georg Glasze. “Who Maps Middle Eastern Geographies in the Digital Age?: Inequalities in Web 2.0 Cartographies in Israel/Palestine.” Media and Mapping Practices in the Middle East and North Africa: Producing Space, edited by Alena Strohmaier and Angela Krewani, Amsterdam University Press, Amsterdam, 2021, pp. 45–70. JSTOR, www.jstor.org/stable/j.ctv1hw3z0w.6. Accessed 12 Apr. 2021.

Cosgrove, Denis. “Cultural Cartography: Maps and Mapping in Cultural Geography / Les Cartes Et La Cartographie En Géographie Culturelle.” Annales De Géographie, vol. 117, no. 660/661, 2008, pp. 159–178. JSTOR, www.jstor.org/stable/23457601. Accessed 12 Apr. 2021.

“Drawing the Line.” Mobile Mapping: Space, Cartography and the Digital, by Clancy Wilmott, Amsterdam University Press, Amsterdam, 2020, pp. 167–184. JSTOR, www.jstor.org/stable/j.ctvx8b7zc.10. Accessed 12 Apr. 2021.

Gorayeb, Adryane, et al. “JLAG Perspectives: Videography for Participatory Cartography in a Site of Wind Power Conflict in Coastal Ceará State, Brazil.” Journal of Latin American Geography, vol. 16, no. 3, 2017, pp. 159–163. JSTOR, www.jstor.org/stable/44861401. Accessed 11 Apr. 2021.

Gutiérrez, Miren. “How Interactive Maps Mobilize People in Geoactivism.” Data Visualization in Society, edited by Martin Engebretsen and Helen Kennedy, Amsterdam University Press, Amsterdam, 2020, pp. 441–456. JSTOR, www.jstor.org/stable/j.ctvzgb8c7.32. Accessed 12 Apr. 2021.

“Introducing Geocriticism.” Topophrenia: Place, Narrative, and the Spatial Imagination, by Robert T. Tally, Indiana University Press, Bloomington, Indiana, USA, 2019, pp. 36–52. JSTOR, www.jstor.org/stable/j.ctv7r40df.6. Accessed 12 Apr. 2021.

**Comment by vcta99** [comment]
Holy cow, Jason. You win all the mint-y pun rewards. I'm not sure if I have much of substance to add, other than I am completely blown away by how thorough and valuable your thinking cap this week was. The nuance of Part 3 of your response (Participatory Mapping and Critical Cartography in Modern Lenses: Israel and Palestine) will specifically stick with me. The OSM nodes vs population based mapping reminded me a lot of gerrymandering techniques in US congressional districts, and the added nuance of dishonest or biased participation in participatory mapping made me reconsider how tools of empowerment can sometimes be abused.

**Comment by epham36** [comment]
This was such a thoughtful and in-depth response! Ever since taking Asian Am 175B with Professor Qutami (highly, highly recommend if it's being offered again!! One of my absolute most rewarding classes at UCLA), I've been fascinated with the Palestine-Israel question. 

You close out your write-up with an interesting question: "Perhaps the solution would be to "take the middle ground" of that data point, but that would also lead to the argument of: what exactly is the middle ground and who gets to define it?" Objectivity is often a myth -- one that encourages the hegemony and bias of one view over all others. Participatory mapping, a method that places a premium on the vantage point of individual people, is inherently subjective. To search for an objective middle ground when maps and people inherently have opinions, may be an impossible or misguided task. 

Part of me wants to use mapping to dive even deeper into the histories and the historical context behind how divisive relations in this region is, especially as it relates to the formation of encampments, borders, and blockades. Definitely, there is power to a bottom-up approach to mapping that would lend an eye-opening, and perhaps more representative, depiction of this region -- as opposed to academicized top-down approaches to mapping. This makes me wonder how mapping and participatory mapping may have been used for activism in Israeli-occupied territories or during the Arab Spring, when there was significant grassroots mobilization in the MENA region as a different, yet geographically relevant use of participatory mapping. I'm certain participatory mapping may have played a role at a more tangible level when activists were orchestrating demonstrations, highlighting blockades, or distributing mutual aid in Palestine and throughout the MENA region.

**Comment by avaitt** [comment]
Love your report! It's so thorough. I really was intrigued by how you closed your report when you mentioned: "Perhaps the solution would be to "take the middle ground" of that data point, but that would also lead to the argument of: what exactly is the middle ground and who gets to define it?" I think a middle ground strikes some sort of data or visualization that is both standardized, accurate and takes into account the sensitivity surrounding a certain issue. I really enjoyed your take on Part 3. The OSM nodes vs. population mapping really helped shine a light on inequality and injustice in areas other than the United States. Now more than ever, I feel as though the world revolves around issues in the United States, but it gives a new perspective when we are able to see trends in mapping that appear around the world.
